Dockerfile.dev

```diff
@@ -0,0 +1,32 @@
+ARG RUBY_VERSION=3.3.0-bullseye
+FROM ruby:$RUBY_VERSION
+
+RUN apt-get update -qq && \
+  apt-get install -y \
+  build-essential \
+  git \
+  libvips \
+  bash \
+  bash-completion \
+  libffi-dev \
+  tzdata \
+  postgresql \
+  nodejs \
+  npm \
+  yarn && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/* /usr/share/doc /usr/share/man
+
+# Comment all of this if you do not wish to install neovim
+RUN apt-get install -y --no-install-recommends curl wget xz-utils && \
+  gem install solargraph:0.50.0 rubocop:1.59.0 neovim:0.9.1 bundler:2.5.3 && \
+  wget -q https://github.com/neovim/neovim/releases/download/stable/nvim-linux64.tar.gz && \
+  tar -xf nvim-linux64.tar.gz && mv nvim-linux64 nvim && \
+  printf "export PATH=/nvim/bin:\$PATH" >> ~/.bashrc && \
+  git clone -b only_ruby https://github.com/RickHPotter/hazyvim.nvim.git ~/.config/nvim
```

Comment:
The branch only_ruby was created in my hazy_nvim.git to make things ligther on the Docker. I did not test it enough to see how good (or bad) it was though.